bitnami/etcd/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.6.4
+version: 4.7.4
```

Comment:
minor version according to semver would be `4.7.0`

Reply:
you're right, sorry 🤦‍♂